sync3/lists_test.go

```diff
@@ -41,7 +41,7 @@ func addRooms(list *sync3.InternalRequestLists, n int) {
 			UserRoomData: caches.UserRoomData{
 				IsDM: next%10 == 0,
 			},
-			LastInterestedEventTimestamp: messageTimestamp,
-		}, true)
+			LastInterestedEventTimestamps: make(map[string]uint64),
```

Comment:
Maybe I should build this properly to be the map from the keys of `list` to `messageTimestamp`? Can't remember why I didn't do so...

Reply:
Oh, I remember the problem. I can't get my hands on the list keys.

Reply:
5f3c9df